### DISCUSSION TITLE
Query: how to change the default order of the geoman tool buttons

### DISCUSSION THREAD

**Discussion by CityBelfast** [geoman-io/leaflet-geoman#851] [Questions & Help]
Hi -  if user wants to set the buttons in the following order is it possible to rearrange built-in tool buttons by passing sort order number or index number?. how can we achieve it 
1. Polygon 
2. Circle
3. Rectangle
3. Marker
4. Line

Here is the default order.
![image](https://user-images.githubusercontent.com/76213912/105369380-676bc600-5bfa-11eb-817d-94316a769937.png)

**Comment by Falke-Design** [comment]
use this: `map.pm.Toolbar.changeControlOrder(["drawPolygon","drawCircle","drawRectangle","drawMarker","drawPolyline"])`

The names are from here: https://github.com/geoman-io/leaflet-geoman#leaflet-geoman-toolbar

**Comment by CityBelfast** [comment]
Thanks @Falke-Design ..it's really help full.
